src/main/kotlin/no/nav/klage/oppgave/api/controller/KvalitetsvurderingController.kt

```diff
@@ -19,7 +18,8 @@ class KvalitetsvurderingController(
     val klagebehandlingService: KlagebehandlingService
 ) {
 
-    @PostMapping("/kvalitetsvurdering/manuell")
+    //TODO Needs new implementation
+//    @PostMapping("/kvalitetsvurdering/manuell")
```

Comment:
Er dette noe som ikke brukes nå?

Reply:
Stemmer. Noe som er laget for fremtidig bruk.